### ISSUE TITLE
Add sorting options to projects

### ISSUE THREAD

**Issue by nothingTVatYT** [MineBill/Seed#6]
I tend to create a new test project for an issue or repro and so the projects overview can get crowded.

A widget for sorting options in the projects tab would be nice, e.g. sort by name and date of last change, ascending and descending.
The cherry on top would be if the chosen sorting would be stored in the user's preferences.

**Comment by MineBill** [comment]
Along with the redesign, i'll be implementing a search bar for the project view. It should make searching for a project faster until this properly gets implemented as a feature.

**Comment by nothingTVatYT** [comment]
The sorting by date does not do anything yet - at least here.

https://github.com/MineBill/Seed/assets/34131388/23631239-3304-4eac-9d49-8871a424f33b

**Comment by MineBill** [comment]
Yeah, i forgot to mention that up until this moment the last time you opened the projects wasn't actually saved but it is now. So from now on, opening a project should correctly update it's last open time and you should see it immediately sorted.

**Comment by nothingTVatYT** [comment]
Ah, I see. OK.

**Comment by nothingTVatYT** [comment]
I tested it and I think it should take the modification time of the project rather than the time you clicked on a project for sorting.
That would solve two effects:
a) the initial sorting is quite random
b) as soon as you open a project that is not the first in the list it moves away under the mouse cursor

**Comment by nothingTVatYT** [comment]
The modification time of the .flaxproj reflects the modification of the project IMO. If I click on a project it's not necessarily modified so to my understanding of sorting by modification time it should take the file modification time of the project file. That would mean some asynchronous check (polling) because it's going to be changed outside of the control of Seed though.
The immediate reordering was surprising to me. I didn't expect the project that I clicked to move away so soon and as a result of the double click. It's just something people may not be used to and be surprised. Using the modification time asynchronously would solve that because the reordering would happen when the user saves changes to the project in the flax editor rather than immediately on a mouse action in Seed. It would feel like reordering is a consequence of a modification action of the project rather than an action in Seed.
